src/index.jsx

```diff
@@ -119,6 +123,7 @@ ReactAudioPlayer.defaultProps = {
   preload: 'metadata',
   src: null,
   style: {},
+  title: ''
```

Comment:
This should have a trailing comma by the linting rules.  I'm going to go ahead and merge this and fix the commas in a subsequent commit myself.